pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/participant/ParticipantDao.java

```diff
@@ -64,4 +68,25 @@ public Optional<ParticipantDto> get(long id) {
         return Optional.empty();
     }
 
+    public Optional<String> getParticipantFromCollaboratorParticipantId(String participantId) {
```

Comment:
The parameter name should be `collaboratorParticipantId` instead of `participantid` I believe, to avoid confusion by what we are fetching the record.